ClientApp/src/components/landingPage/Contact.tsx

```diff
@@ -15,9 +15,67 @@ import useStyles from "./Styles";
 import dummyImg from "./../../styling/img/dummy-image.jpg";
 import { Mail, Facebook, Instagram } from "@material-ui/icons";
 
-const Contact = () => {
+
+export interface ContactsData {
```

Comment:
The interface name should not be plural, as it represents a single contact.
Eg. ContactData or ContactInfo should ble more fitting.